### PR TITLE
Update perfdash README with deployment instructions

### PR DESCRIPTION
**Pull request by pierewoj**

### CODE REVIEW

File: perfdash/README.md [hunk 1]
```diff
@@ -22,6 +22,21 @@ Metrics above are available for all kinds of tests divided into load and density
 Application server runs as a deployment on kubernetes cluster. It is hosted on
 *mungegithub* cluster in *k8s-mungegithub* project.
 
+## How to deploy new version of perf-dash.k8s.io
+
+* Increment TAG in the `Makefile` (for example: 2.10 -> 2.11)
+* Modify the perfdash image version in the `deplyoment.yaml` to be the same as the
+  one specified in the `Makefile`
+* Submit a PR, get required approvals and wait until it's merged
+* In GCP console, select `k8s-mungegithub` project
```
Comment (mm4tt) [inline perfdash/README.md]: Can we squash these two lines (this and one below) into: 
* Configure kubectl to connect to the K8s cluster that hosts perf-dash (`mungegithub` cluster in the `k8s-mungegithub` project).

File: perfdash/deploy.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euxo pipefail
+
+KUBECONFIG=$HOME/.kube/config-tmp-mungegithub-$(date +%F_%H_%M_%S)
```
Comment (mm4tt) [inline perfdash/deploy.sh]: NIT: I'd do export KUBECONFIG=$HOME/...

Comment (pierewoj) [reply perfdash/deploy.sh]: https://github.com/koalaman/shellcheck/wiki/SC2155

Comment (mm4tt) [reply perfdash/deploy.sh]: Make sense, thanks

**Pull request merged**